pyzeebe/channel/utils.py

```diff
@@ -4,9 +4,152 @@
+        str: The client secret.
+            Default: Value from CAMUNDA_CLIENT_SECRET or ZEEBE_CLIENT_SECRET environment variable
+    """
+    ret = client_secret or os.getenv("CAMUNDA_CLIENT_SECRET", os.getenv("ZEEBE_CLIENT_SECRET"))
+
+    if not ret:
+        raise ValueError(
+            "parameter client_secret or one of the environment variables CAMUNDA_CLIENT_SECRET or ZEEBE_CLIENT_SECRET must be provided!"
+        )
+
+    return ret
+
+
+def get_camunda_cloud_cluster_id(cluster_id: Optional[str]) -> Optional[str]:
+    """
+    Args:
+        cluster_id (str, optional): The camunda cluster id provided as parameter.
+
+    Returns:
+        str: The camunda cluster id.
+            Default: Value from CAMUNDA_CLUSTER_ID environment variable
+    """
+
+    return cluster_id or os.getenv("CAMUNDA_CLUSTER_ID")
+
+
+def get_camunda_cloud_cluster_region(cluster_region: Optional[str]) -> str:
+    """
+    Args:
+        cluster_region (str, optional): The camunda cluster region provided as parameter.
+
+    Returns:
+        str: The camunda cluster region.
+            Default: Value from CAMUNDA_CLUSTER_REGION environment variable or 'bru-2'
+    """
+
+    return cluster_region or os.getenv("CAMUNDA_CLUSTER_REGION") or "bru-2"
+
+
+def get_camunda_token_audience(token_audience: Optional[str]) -> Optional[str]:
+    """
+    Args:
+        token_audience (str, optional): The token audience provided as parameter.
+
+    Returns:
+        str: The token audience.
+            Default: Value from CAMUNDA_TOKEN_AUDIENCE
+                     or ZEEBE_TOKEN_AUDIENCE environment variable
+                     or camunda cloud token audience if camunda cluster_id is available
+    """
+
+    return (
+        token_audience
+        or os.getenv("CAMUNDA_TOKEN_AUDIENCE", os.getenv("ZEEBE_TOKEN_AUDIENCE"))
+        or get_camunda_cloud_hostname(None, None)
+    )
+
+
+def get_camunda_cloud_hostname(cluster_id: Optional[str], cluster_region: Optional[str]) -> Optional[str]:
+    """
+    Args:
+        cluster_id (str, optional): The camunda cluster id provided as parameter.
+        cluster_region (str, optional): The camunda cluster region provided as parameter.
+
+    Returns:
+        str: The token audience for camunda cloud or none if cluster_id or cluster_region is not provided.
+    """
+
+    cluster_id = get_camunda_cloud_cluster_id(cluster_id)
+    cluster_region = get_camunda_cloud_cluster_region(cluster_region)
+
+    if (not cluster_id) or (not cluster_region):
+        return None
+
+    return f"{cluster_id}.{cluster_region}.zeebe.camunda.io"
+
+
+def get_camunda_credentials_scopes(scopes: Optional[str]) -> Optional[str]:
+    """
+    Args:
+        scopes (str, optional): The camunda credentials scopes as single string.
+
+    Returns:
+        str: The camunda cedential scopes string.
+            Default: Value from CAMUNDA_CREDENTIALS_SCOPES
+    """
+
+    return scopes or os.getenv("CAMUNDA_CREDENTIALS_SCOPES")
```

Comment:
`CAMUNDA_CREDENTIALS_SCOPES='Zeebe' is the scope of the Camunda Console credentiels, **not** the scope for authorization.

This is not to be used.

Reply:
Do you know about some documentation regarding this?

Reply:
According to the old code at https://github.com/camunda-community-hub/pyzeebe/pull/509/files#diff-24bc55faae9e3f2a8af987d3894672f979e646328ccd96c67b71f308a02f3e37L83 `Zeebe` was previously also used as the scope. So I am not sure what you mean.

Reply:
I can't find it but the Camunda Docs only mention audience not scope and applying the scope to `CAMUNDA_CREDENTIALS_SCOPES` aka `Zeebe` fails with my own integration tests. #514 #511 #510

Reply:
I have removed the default value for the scopes parameters